### ISSUE TITLE
Problems with skill

### ISSUE THREAD

**Issue by Larson-Sei180LX** [NorthernMan54/homebridge-alexa#151]
Hey there, 

i installed that plugin with npm install -gs homematic-alexa in docker container. No errors, but it doesnt work.cant find any devices. Does that plugin still work with hombridge skill from alexa?

**Comment by NorthernMan54** [comment]
Yes it still works and there are thousands using it on a regular basis.  If you look at the logs, it is pretty descriptive of the issues, and if enough detail isn’t provided, you can run it with the DEBUG flag to supply more details.

**Comment by Larson-Sei180LX** [comment]
Could someone please check my config.json - its only a part of it - i tried it with ssdp and without - nothing is happening...



{
	"bridge": {
		"name": "HomeBridge",
		"username": "CC:22:3D:E3:CE:30",
		"port": 51826,
		"pin": "031-45-154",
	        "ssdp": 1900
	},
	"platforms": [{
		"platform": "Alexa",
		"name": "Alexa",
		"username": "xxx@yyy.de",
		"password": "testtest",
		"refresh": 15
	}],

**Comment by Larson-Sei180LX** [comment]
this is my Accountname there..perhaps u could check if it is linked ? LarsonSei180LX
